tests/Doctrine/Migrations/Tests/Tools/Console/Command/ExecuteCommandTest.php

```diff
@@ -31,7 +31,7 @@ public function testWriteSqlCustomPath() : void
 
         $input->expects(self::once())
             ->method('getArgument')
-            ->with('version')
+            ->with('versions')
```

Comment:
What do you think about adding tests that demonstrate that this actually works with

- one specified version
- multiple specified versions

?